src/components/NavigationSection.svelte

```diff
@@ -69,6 +76,22 @@
     }
   }
 
+  function getCategories (items) {
+    const categoriesOject = {}
+    items.forEach(i => {
```

Comment:
So you use the standard Interger var name as an object ? Isn’t this error prone ?